android/app/src/main/java/app/organicmaps/util/Config.java

```diff
@@ -357,6 +357,13 @@ public static String getDonateUrl(@NonNull Context context)
     return url;
   }
 
+  @NonNull
+  @SuppressWarnings("ConstantConditions") // BuildConfig
+  public static boolean isOsmLoginEnabled(@NonNull Context context)
+  {
+    return !BuildConfig.FLAVOR.equals("google") || !getString(KEY_DONATE_URL).isEmpty();
```

Comment:
What is the role of the `KEY_DONATE_URL` condition?

Reply:
Should remove this condition.